docs/conf.py

```diff
@@ -24,7 +24,7 @@
 author = "Earth Lab"
 
 # The short X.Y version
-version = ""
+version = "0.6.3"
```

Comment:
is there a way to have this updated based upon the setup file @mbjoseph  ? otherwise it's a manual update. it doesn't match because i just bumped the version after our last meeting on wednesday and this PR has been open.

Reply:
Yes @lwasser we should be able to automate this. Check out the `setup.cfg` file for an example of how bumpversion finds and replaces the version in `setup.py`. I'm guessing you could add a similar chunk to that file, e.g. something like

```cfg
[bumpversion]
current_version = 0.6.5
commit = True
tag = True

[tool:pytest]
testpaths = earthpy/tests
filterwarnings = 
	ignore::rasterio.errors.NotGeoreferencedWarning

[bumpversion:file:setup.py]
search = "version='{current_version}'"
replace = {new_version}

[bumpversion:file:docs/conf.py]
search = "version = '{current_version}'"
replace = {new_version}
```

For more examples see: https://github.com/search?q=bumpversion%3Afile%3Adocs%2Fconf.py&type=Code

Reply:
ok i'll give that a go.

Reply:
it worked!!